doc/src/man/litani-dump-run.scdoc

```diff
@@ -22,12 +22,13 @@ litani dump-run - Print the current run as a JSON document to stdout
 # SYNOPSIS
 
 *litani dump-run*
-	\[*--retries* _N_]
+	\[*-r*/*--retries* _N_]
+	\[*-o*/*--out-file* _F_]
 
 
 # DESCRIPTION
 
-This program prints the _run.json_ file for an in-progress Litani run to stdout.
+This program prints the run.json file for an in-progress Litani run to stdout or to the specified output file.
```

Comment:
`in-progress` can be omitted. `dump-run` can print the run.json after a Litani build has concluded

Reply:
Resolved